src/phoenix/server/main.py

```diff
@@ -104,22 +119,27 @@ def _get_pid_file() -> Path:
             reference_dataset = None
     elif args.command == "trace-fixture":
         trace_dataset_name = args.fixture
+        simulate_streaming = args.simulate_streaming
 
     model = create_model_from_datasets(
         primary_dataset,
         reference_dataset,
     )
     traces = Traces()
     if trace_dataset_name is not None:
-        for span in map(
+        fixture_spans = map(
             json_string_to_span,
             _download_traces_fixture(
                 _get_trace_fixture_by_name(
                     trace_dataset_name,
                 ),
             ),
-        ):
-            traces.put(span)
+        )
+        Thread(
+            target=_load_spans,
+            args=(traces, fixture_spans, simulate_streaming),
+            daemon=True,
+        ).start()
```

Comment:
Does running as a thread have any impact resource wise in colab? Just checking.

Reply:
There shouldn't be. Incidentally, this code path only gets run by `ProcessServer` which spawns a process and this thread will be inside that process.

Reply:
Since it's an iterator, we can also put `cycle()` on it and make it infinite...